fourkitchens/emulsify
# PL Trans tag:  Fixes for 214

Continuation of work started on https://github.com/fourkitchens/emulsify/pull/214

How to test:
1. Checkout branch and run `yarn start` (ensure no errors)
2. Using this branch as your theme in a Drupal installation, create some content.
3. Notice that on this branch the authoring info changes from `by @author_name @created_date` to something like `by admin Mon, 01/29/2018 - 19:21`

**Before:**
<img width="281" alt="screen shot 2018-05-30 at 10 47 55 am" src="https://user-images.githubusercontent.com/18293479/40731565-f1fc6c7e-63f6-11e8-82a0-5a6ab031c5cc.png">

**After:**
<img width="268" alt="screen shot 2018-05-30 at 10 47 27 am" src="https://user-images.githubusercontent.com/18293479/40731575-f64c3444-63f6-11e8-92c4-d7e45605c89a.png">
